models/userDao.js

```diff
@@ -0,0 +1,66 @@
+const appDataSource = require('./appDataSource');
+
+const signUp = async (first_name, last_name, email, password) => {
```

Comment:
```suggestion
const createUser = async (first_name, last_name, email, password) => {
```
함수 네이밍, 특히 dao 함수는 실질적인 내부 로직 및 동작과 연관지어 작성해주시는것이 가장 직관적입니다. 이때는 CRUD를 의미하는 create, get, update, delete 와 목적자원의 조합으로 짓는것이 일반적입니다.